package.json

```diff
@@ -75,9 +88,9 @@
     "highlight.js": "^8.6.0",
     "marked": "^0.3.3",
     "mocha": "^2.2.5",
-    "page": "^1.6.3",
+    "page": "1.6.3",
     "react": "^0.13.3",
-    "react-bootstrap": "^0.23.5",
+    "react-bootstrap": "0.23.5",
```

Comment:
I pinned down the `react-bootstrap` version here because the current examples (which I have not really touched) break on the latest version.